File: backend/src/main/java/com/kiwi/features/quests/data/SubquestDTO.java
package com.kiwi.features.quests.data;

import lombok.*;

@Getter @Setter @NoArgsConstructor @AllArgsConstructor @Builder @EqualsAndHashCode @ToString
public class SubquestDTO {
    private int subquestId;
    private String name;
    private int experience;
    private int order;
    private String status;
    private String onCompletedEvent;
    private int onCompletedEntityId;
}

File: mobile/app/src/main/java/com/bellako/kiwi/features/quests/data/SubquestDTO.kt
package com.bellako.kiwi.features.quests.data

data class SubquestDTO(
    val subquestId: Int,
    val name: String,
    val experience: Int,
    val order: Int,
    val status: String,
    val onCompletedEvent: String,
    val onCompletedEntityId: Int,
)
